modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,182 @@
+----
+NAME     STATUS STATE                  CONSUMER            BMC                    HARDWARE PROFILE ONLINE ERROR
+master-1 OK     externally provisioned kni1-master-1       ipmi://172.22.0.1:6231                  true
+master-2 OK     externally provisioned kni1-master-2       ipmi://172.22.0.1:6232                  true
+master-3 OK     available                                  ipmi://172.22.0.1:6233 unknown          true
+worker-0 OK     provisioned            kni1-worker-0-kjpmp ipmi://10.19.143.29    dell-raid        true
+----
++
+There are no `MachineSet` objects for control plane nodes, so you must create a `MachineSet` object instead. You can copy the image URL and checksum from the other `BareMetalHost` objects.
+
+. Create a `MachineSet` object:
++
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: machine.openshift.io/v1beta1
+kind: Machine
+metadata:
+  annotations:
+    metal3.io/BareMetalHost: openshift-machine-api/ocp-master-2
+  labels:
+    machine.openshift.io/cluster-api-cluster: kni1
+    machine.openshift.io/cluster-api-machine-role: master
+    machine.openshift.io/cluster-api-machine-type: master
+  name: kni1-master-2
+  namespace: openshift-machine-api
+spec:
+  metadata: {}
+  providerSpec:
+    value:
+      apiVersion: baremetal.cluster.k8s.io/v1alpha1
+      customDeploy:
+        method: install_coreos
+      hostSelector: {}
+      image:
+        checksum: ""
+        url: ""
+      kind: BareMetalMachineProviderSpec
+      metadata:
+        creationTimestamp: null
+      userData:
+        name: master-user-data-managed
+EOF
+----
++
+. To define and create the `BareMetalHost`, `Secret`, and `MachineSet` objects in a single YAML file, run the following command:
++
+[source,terminal]
+----
+$ oc create -f example.yaml
+----
++
+The provisioning process uses the baremetal-operator to install RHCOS and prepare the host to be added to the cluster. 
++
+. To view the `baremetalhost` objects, run the following command:
++
+[source,terminal]
+----
+$ oc get bmh -A
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE               NAME           STATUS PROVISIONING STATUS     CONSUMER        BMC                     HARDWARE PROFILE   ONLINE   ERROR
+openshift-machine-api   kni1-master-0  OK     provisioned             kni1-worker-0   ipmi://10.19.136.22     dell-raid          true
+openshift-machine-api   master-1       OK     externally provisioned  kni1-master-1   ipmi://172.22.0.1:6231                     true
+openshift-machine-api   master-2       OK     externally provisioned  kni1-master-2   ipmi://172.22.0.1:6232                     true
+openshift-machine-api   master-3       OK     provisioning            kni1-master-3   ipmi://172.22.0.1:6233  unknown            true
+openshift-machine-api   worker-0       OK     provisioned             kni1-worker-1   ipmi://10.19.143.29     dell-raid          true
+----
++
+. Verify that the `BareMetalHost` is added to the cluster:
++
```

Comment:
There is no need to show this output, it would be the same than the previous one.

Reply:
Removed output as suggested.